src/components/Main/Main.test.js

```diff
@@ -0,0 +1,74 @@
+import { toBeInTheDocument } from "@testing-library/jest-dom/dist/matchers";
+import {
+  findByAltText,
+  getByAltText,
+  getByText,
+  render,
+  screen,
+  waitFor,
+} from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import moment from "moment";
+import Main from "./Main";
+
```

Comment:
We may need to reuse asserts through each acceptance criteria, so a shared assert function is a nice way to ensure Picture of the day is displaying correclty.
```suggestion
const mockedImageOfTheDayRendered = async () => {
  const image = await waitFor(() =>
    screen.getByAltText("Astronomy Picture of the Day")
  );
  expect(image).toBeInTheDocument();
  expect(image).toHaveProperty(
    "src",
    "https://apod.nasa.gov/apod/image/2205/M87bh_EHT_960.jpg"
  );
  expect(
    screen.getByText(/What does a black hole look like/i)
  ).toBeInTheDocument();
};
```